src/fscacher/cache.py

```diff
@@ -21,12 +21,18 @@ class PersistentCache(object):
 
     _cache_var_values = (None, "", "clear", "ignore")
 
-    def __init__(self, name=None, tokens=None, envvar=None):
+    def __init__(self, name=None, *, path=None, tokens=None, envvar=None):
```

Comment:
Something to check in places where we used either tokens are specified via kwarg